wordpress-mobile/WordPress-Android
# Stats:  update default insights list

This PR brings in [changes done on FluxC](https://github.com/wordpress-mobile/WordPress-Android/issues/16029) to update default insights list

Fixes #16029 

To test:

Test 1

- Install a previous version than this branch (e.g. trunk)
- Choose a site where you have not customized the Insights screen in Stats
- Navigate to My Site -> Stats -> Insights
- Verify that the default cards are: "Latest Post Summary", "Today", "All Time" and "Follower Totals", plus any pinned cards (e.g. "Grow Audience or "Customize")

Test 2

- Use this build
- Navigate to My Site -> Stats -> Insights
- Verify that the default cards now are: "Most Popular Time", "All Time", "Today", "Followers", "Comments"
- Repeat the steps with several sites and observe the behavior: previously customized pages should be left as they were


## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested with different site and combinations

3. What automated tests I added (or what prevented me from doing so)
Updated unit tests on FluxC part

PR submission checklist:
